Library/Formula/packer.rb

```diff
@@ -0,0 +1,224 @@
+require "language/go"
+
+class Packer < Formula
+  homepage "http://packer.io"
```

Comment:
I didn't update the PR (because I didn't want to regenerate the bottles, which is slow); I amended the commit as I merged. It's in 68a0786d0c5a4848d79d3faae6a1afae488e4250.